pkg/loadbalancer/experimental/writer.go

```diff
@@ -396,16 +396,17 @@ func (w *Writer) UpsertBackends(txn WriteTxn, serviceName loadbalancer.ServiceNa
 
 // SetBackends sets the backends associated with a service. Existing backends from this source that
 // are associated with the service but are not given are released.
-func (w *Writer) SetBackends(txn WriteTxn, name loadbalancer.ServiceName, source source.Source, bes ...BackendParams) error {
+func (w *Writer) SetBackends(txn WriteTxn, name loadbalancer.ServiceName, source source.Source, clusterID uint32, bes ...BackendParams) error {
```

Comment:
It could make sense to validate that the `clusterID` passed as parameter matches the one set in `BackendParams` (or maybe forcefully overwrite it?). Funky things could otherwise happen if they don't match.

Reply:
I'll just forcefully overwrite it the same as with Source.